benchexec/tools/sesl.py

```diff
@@ -0,0 +1,42 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    A Symbolic Executor based on Separation Logic
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("sesl-svcomp.sh")
+
+    def name(self):
+        return "SESL"
+
+    def version(self, executable):
+        return self._version_from_tool(executable, arg="--version")
+
+    def cmdline(self, executable, options, task, rlimits):
+        options += ["-t", "--sh-mem-leak", "--add-line-info"]
+        if task.property_file:
+            options += ["--svcomp-property", task.property_file]
+        options += list(task.input_files_or_identifier)
```

Comment:
Does your tool accept tasks with more than one input file or without any input file? If not, I suggest to use `task.input_files` or `task.single_input_file`, because these will produce proper error messages for the user in case an unsupported task is used.